libraries/AP_Scripting/drivers/LTE_modem.lua

```diff
@@ -1110,7 +1144,9 @@ end
 --]]
 local function check_QENG(s)
     -- Example1: +QENG: "servingcell","NOCONN","LTE","FDD",505,02,12AED4A,445,3750,8,3,3,CBE8,-99,-14,-71,53,30
-    -- Example2: +QENG: "servingcell","NOCONN","LTE","FDD",505,02,22D3F32,271,9260,28,3,3,CBE8,-109,-15,-78,38,20
+   -- Example2: +QENG: "servingcell","NOCONN","LTE","FDD",505,02,22D3F32,271,9260,28,3,3,CBE8,-109,-15,-78,38,20
+   -- Example3: +QENG: "servingcell","CONNECT","eMTC","FDD",505,01,804A90D,238,9410,28,5,5,2036,-114,-18,-82,6
+
```

Comment:
Modifying tabs?

Reply:
I don't understand how, this just makes it not line up with the rest of the code.

Reply:
I've changed it, but can we please not hold up this PR over trivialities like this? I'm trying to help users get this working and holding up over formatting of a comment is just annoying